doc/usecases/snippet_sharing/UC_convert_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Convert configuration snippet
+
+## Summary
+
+Title: Convert configuration snippet
+Scope: Configuration conversion
+Level: User Goal
+Actors: Anonymous user (+ all others)
+Brief: User can input a configuration snippet in any supported format (e.g. ini, xml) and convert it into another supported format (e.g. json).

```

Comment:
As discussed: Maybe its better to have this as front-end feature and not as API feature?